src/main/kotlin/view/WinningStatisticsView.kt

```diff
@@ -31,4 +32,13 @@ object WinningStatisticsView {
     ) {
         println("총 수익률은 ${statistics.calculateRatioOfIncomeToExpenditure(lottoPrice).value}입니다.")
     }
+
+    private fun WinningCategory.toDescription(): String = when (this) {
+        WinningCategory.SIX_CORRECT -> "6개 일치"
+        WinningCategory.FIVE_WITH_BONUS_CORRECT -> "5개 일치, 보너스 볼 일치"
+        WinningCategory.FIVE_CORRECT -> "5개 일치"
+        WinningCategory.FOUR_CORRECT -> "4개 일치"
+        WinningCategory.THREE_CORRECT -> "3개 일치"
+        else -> "꽝"
```

Comment:
`else`를 쓰지 않고도 가능합니다.

```suggestion
        WinningCategory.LOSE -> "꽝"
```

Reply:
enum class, sealed class와 when expression은 상성이 좋습니다.
switch 문에 무조건 default를 써줘야했던 자바와 다르게 코틀린은 컴파일러가 똑똑해서 합타입(enum, sealed class)의 경우에는 else를 생략할 수 있습니다.